doc/fileexplorer.rst

```diff
@@ -1,28 +1,79 @@
-:orphan:
+#####
+Files 
+#####
 
-#############
-File Explorer
-#############
-
-The **File Explorer** pane is a built-in filesystem and directory browser allowing you to view and filter files and their properties, open them with the :doc:`editor` or an external tool, and perform common management tasks (varies by platform).
+The **Files** pane is a built-in filesystem and directory browser that allows you to view and filter files according to their type and extension.
+You can also open files with the :doc:`editor` or an external tool, and perform some operations.
 
 .. image:: images/file_explorer/file_explorer_standard.png
    :alt: Spyder File Explorer panel, showing a tree view of files and metadata
 
 |
 
 
-==========================
-Performing file operations
-==========================
 
-You can use context menus to access a number of functions on a per-file and per-directory basis, including running scripts, creating, renaming, moving, editing or deleting files, performing basic operations with ``git``, and (on Windows) opening an external file explorer.
+===============
+File Operations
+===============
+
+To browse between the files in your system, you can use the arrows in the top of the pane.
+You can expand/contract the folders displayed in the pane to deploy the files and subfolders hierarchically. 
+Double cliking a folder will open it showing the files on it and changing your working directory.
+
+[GIF showing browsing files and showing changing the working directory]
+
+To open a file in the editor from the file pane, double click over its name.
+If you right click over it, you will see a context menu that allows you to access a number of functions including running scripts, creating, renaming, moving, deleting and showing files in your computer's file manager.
+
+[Screenshot showing context menu]
+
+You can also copy and paste one or several files in the folder where you are located in the pane, or copy their absolute and relative's path in the clipboard to paste it as text.
```

Comment:
```suggestion
You can copy and paste one or several files to and from the pane, and copy their absolute or relative paths to the clipboard as text.
If copying the paths for multiple files, they will be automatically formatted so you can paste them directly into a Python list.
```

Fix grammar and wording errors and simplify/clarify description, I found it rather hard to understand (though I'm still not entirely happy with the rewording...suggestions more than welcome). I also added a description of a particularly useful part of this feature that users might not be initially aware of.